commands.go

```diff
@@ -26,8 +28,72 @@ func (cmd *SortCommand) Command() *imap.Command {
 	}
 }
 
+func parseSortCrit(fields interface{}) ([]SortCriterion, error) {
```

Comment:
Let's not use an abbreviation here: `parseSortCriteria` isn't too long.

Reply:
Still curing my desire to abbreviate names :)